python/datasources/cdc_hiv.py

```diff
@@ -149,14 +189,50 @@ def load_df_from_data_dir(geo_level: str, breakdown: str):
     """
     load_df_from_data_dir fetches the csv file from the data dir
 
-    filename: the name of the file to load the csv file from
-    return: a dataframe for specified geo_level and breakdown
+    breakdown: string equal to `age`, `race_and_ethnicity`, or `sex`
+    geo_level: string equal to `county`, `national`, or `state`
+    return: a dataframe for Diagnoses, Deaths, and PrEP at the specified
+    geo_level and breakdown
     """
-    filename = f'hiv-{geo_level}-{breakdown}.csv'
-    df = gcs_to_bq_util.load_csv_as_df_from_data_dir(HIV_DIR,
-                                                     filename,
-                                                     skiprows=8,
-                                                     thousands=',',
-                                                     dtype={'FIPS': str, 'Year': str})
+    # data for HIV diagnoses is available at every geo level and breakdown
+    diagnoses_df = gcs_to_bq_util.load_csv_as_df_from_data_dir(HIV_DIR,
+                                                               f'{DIAGNOSES}-{geo_level}-{breakdown}.csv',
+                                                               subdirectory=DIAGNOSES,
+                                                               skiprows=8,
+                                                               na_values=NA_VALUES,
+                                                               thousands=',',
+                                                               dtype=DTYPE)
+    df = diagnoses_df.rename(columns=DIAGNOSES_TO_STANDARD)
+    df = df.drop(columns='Indicator')
```

Comment:
You should use constants for repeated strings like `Indicator` and `Multiracial`, as it helps reduce the probabilities of errors.